packages/ubus-lime-fbw/Makefile

```diff
@@ -25,6 +25,7 @@ define Package/$(PKG_NAME)/install
 	$(CP) ./files/* $(1)/
 	@chmod a+x $(1)/usr/libexec/daemon/lime-fbw
 	@chmod a+x $(1)/etc/init.d/ubus-lime-fbw
+  @chmod a+x $(1)/www/cgi-bin/hostname
```

Comment:
AFAIR  some makefile macro should be used instead con plain chmod to install executables @spiccinini do you remember if it is like that?

Reply:
Yes, `INSTALL_BIN` macro does `chmod +x` (equivalent to `chdmo a+x`)

Reply:
This package is using "the trick" using `CP ./files/*` so using `INSTALL_* ` macros goes against that practicity. I don't know if there is a preferred way. I see mixed uses in other packages (from openwrt and libremesh).